api/migrations/0001_initial.py

```diff
@@ -568,6 +572,8 @@ class Migration(migrations.Migration):
                             ("left", "Left the project"),
                             ("expelled", "Expelled from the project"),
                             ("access_revoked", "Access to corporate resources revoked"),
+                            ("gone_on_leave", "Gone on leave"),
+                            ("returned_from_leave", "Returned from leave"),
```

Comment:
we forgot to recreate the 1st migration in #273, right?

Reply:
Yep